File: WithCallback/app/src/main/java/com/ducluanxutrieu/tikiapp/ui/detail/DetailFragment.java
package com.ducluanxutrieu.tikiapp.ui.detail;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.webkit.WebView;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;

import com.ducluanxutrieu.tikiapp.R;

/**
 * An example full-screen activity that shows and hides the system UI (i.e.
 * status bar and navigation/system bar) with user interaction.
 */
public class DetailFragment extends Fragment {
    private WebView webView;
    @Nullable
    @Override
    public View onCreateView(@NonNull LayoutInflater inflater,
                             @Nullable ViewGroup container,
                             @Nullable Bundle savedInstanceState) {
        View view = inflater.inflate(R.layout.fragment_detail, container, false);
        webView = view.findViewById(R.id.wv_detail);
        return view;
    }

    @Override
    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);
        String url = DetailFragmentArgs.fromBundle(getArguments()).getUrl();
        webView.loadUrl(url);
    }
}

File: app/src/main/java/com/example/tikiapp/ui/detail/DetailFragment.kt
package com.example.tikiapp.ui.detail

import android.os.Bundle
import androidx.fragment.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.lifecycle.ViewModelProvider
import androidx.navigation.fragment.navArgs
import com.example.tikiapp.R
import kotlinx.android.synthetic.main.detail_fragment.*

class DetailFragment : Fragment() {
    private val mArgs: DetailFragmentArgs by navArgs()
    private lateinit var viewModel: DeailViewModel

    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        return inflater.inflate(R.layout.detail_fragment, container, false)
    }

    override fun onActivityCreated(savedInstanceState: Bundle?) {
        super.onActivityCreated(savedInstanceState)
        viewModel = ViewModelProvider(this).get(DeailViewModel::class.java)
//        wv_detail.settings.javaScriptEnabled = true;

        wv_detail.loadUrl(mArgs.url)
    }

}